recipes/alertmanager.rb

```diff
@@ -65,6 +65,9 @@
 
 include_recipe "prometheus::alertmanager_#{node['prometheus']['alertmanager']['install_method']}"
 
+node['prometheus']['flags']['alertmanager.url'] = 'http://127.0.0.1/alert-manager/' if \
+  node['prometheus']['flags']['alertmanager.url'].nil? || node['prometheus']['flags']['alertmanager.url'].empty?
+
```

Comment:
Hi ElijahCaine,

  I'm a little worried about your code placement.  This looks like you want to set a default.  Why not just set the default in the attributes file.

You will also need to specify a level for the value you set.  node.default[....  for instance.

Mark